scripts/issue/156/issue156.py

```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import interface
+
+
+def main():
+    """
+    deposit all bveCVX lp to bbveCVX sett
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+
+    bveCVX = interface.ICurveLP(
```

Comment:
The variable naming was confusing for me, mixing it up with the real `bvecvx` sett token, could this be renamed to reflect the bvecvx_cvx_lp instead?

Reply:
agreed, these token names can get confusing